supplier_app/views.py

```diff
@@ -299,7 +331,7 @@ def _get_company_unique_token_from_token(token):
     return get_object_or_404(CompanyUniqueToken, token=token)
 
 
-@permission_required('users_app.can_approve', raise_exception=True)
+@permission_required(CAN_APPROVE_PERM, raise_exception=True)
```

Comment:
I think having `CAN_APPROVE_PERM` without context it's not a good approach, consider providing context example: `COMPANY_USER_CAN_APPROVE_PERM`

Reply:
Done